### PR TITLE
Add license 

### PR DESCRIPTION
**Pull request by tanvimoharir**
This has the MIT license only(was initially trying to add both the files onto the same branch)

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Pranav016
```
Comment (Pranav016) [inline LICENSE.md:3]: Change `Pranav016` to `Pranav016 & ALPHAVIO organization`

**Review by Pranav016 — CHANGES_REQUESTED** [review]

**Comment by Pranav016** [comment]
I'd recommend you to `squash` your commits for this pull request since you'll be having 3 commits that will be doing the same thing. So squashing all the 3 commits would make sense. Squash them and change their commit message to `Add LICENSE.md`.
Let me know if you face any difficulty.

File: LICENSE.md (hunk 1 above)
Comment (tanvimoharir) [reply LICENSE.md]: This is done.

**Comment by tanvimoharir** [comment]
@Pranav016  I have not squashed any commits before this and I tried to do this on some commits on my private repository,however even after squashing I could still see all the commits.
I'm referred to :
https://levelup.gitconnected.com/how-to-squash-git-commits-9a095c1bc1fc
Let me know if I'm missing something or otherwise.
Thanks!

**Review by Pranav016 — APPROVED** [review]

**Comment by Pranav016** [comment]
@tanvimoharir Great ,now please squash your commits. There are 3 commits that are trying to do the same thing, so we don't want the other two irrelevant commits in the commit history. You can refer to the documentation or a tutorial for squashing, isn't very hard.

**Comment by Pranav016** [comment]
> @Pranav016 I have not squashed any commits before this and I tried to do this on some commits on my private repository,however even after squashing I could still see all the commits.
> I referred to :
> https://levelup.gitconnected.com/how-to-squash-git-commits-9a095c1bc1fc
> Let me know if I'm missing something or otherwise.
> Thanks!

I think this article is correct to some extent but it has missed some small steps that is why you are commits are still visible.
Here are the steps-
```
git rebase -i HEAD~3
```
Now your vim will open up. Press `i` to enter editor mode.
Now let one of the commits remain as `pick` and replace other two picks with `squash` and press `Esc ` and then type ` : wq`.
A new window will open up, press `i` to edit. Then just place `#` in front of the commit messages that are not needed. (Any two)
Retain the commit message that is more relevant.
then press `Esc ` and then type ` : wq`.

That will do the job. These are all the steps easily explained.

**Comment by tanvimoharir** [comment]
@Pranav016 Thanks for the steps. I had followed the same but I thought that the squashed commits won't be visible( as per the link I shared above its supposed to edit git history).
Regardless, I have done the above mentioned steps, Please check and let me know. Thanks again.

**Pull request closed without merging**